de.dlr.sc.virsat.model.extension.cefx/src/de/dlr/sc/virsat/model/extension/cefx/migrator/Migrator1v2.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.extension.cefx.migrator;
+
+import de.dlr.sc.virsat.model.dvlm.concepts.Concept;
+import de.dlr.sc.virsat.model.concept.migrator.IMigrator;
+
+
+// *****************************************************************
+// * Class Declaration
+// *****************************************************************
+
+/**
+ * Auto Generated Class inheriting from Generator Gap Class
+ * 
+ * This class is generated once, do your changes here
+ * 
+ * VirSat DLR CEF Concept for extended Product Structures
+ * 
+ */
+public class Migrator1v2 extends AMigrator1v2 implements IMigrator {
+
+	@Override
+	public void migrate(Concept concept, IMigrator previousMigrator) {
```

Comment:
Please remove this method, otherwise the test coverage will go down although we don't have any implementation in there. But make sure the class is still there ;)

Reply:
I did it.